File: pset6/mario/more/mario.py
# Mario program - more comfortable
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h >= 0 and h <= 23:
        break

# Declaring j and mode variables
j = mode = 0

# For loop on displaying the result
for i in range(1, h + 1):
    mode = 0
    while mode <= 1:
        j = h
        while j > 0:
            # Custom conditions to check what state it is in
            # to output required results
            if mode == 0:
                print(" " if j > i else "#", end="")
            else:
                print("#" if j > h - i else "", end="")
            j -= 1
        print("  " if mode == 0 else "\n", end="")
        mode += 1

File: pset1/mario/more/mario.c
// Mario program - More comfortable
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Height, h and initialise it
    int h = -1;
    while (h < 0 || h > 23)
    {
        h = get_int("Height: ");
    }

    // Declaring j and mode variables
    int j, mode;

    // For loop on displaying the result
    for (int i = 1; i <= h; i++)
    {
        mode = 1;
        while (mode <= 2)
        {
            j = h;
            while (j > 0)
            {
                // Custom conditions to check what state it is in
                // to output required results
                if (mode == 1)
                {
                    printf(j > i ? " " : "#");
                }
                else
                {
                    printf(j > h - i ? "#" : "");
                }
                j--;
            }
            printf(mode == 1 ? "  " : "\n");
            mode++;
        }
    }
}